bin/sl-run.js

```diff
@@ -18,6 +18,30 @@ var agentOptions = {
   logger: config.logger,
 };
 
+var USE_PROBES = true;
+
+if (config.enableTracing) {
+  var tracer = require('../lib/tracer');
+  var traceObject = require('../lib/trace-object');
+  traceObject.tracerOptions.USE_PROBES = USE_PROBES;
+  traceObject.tracerOptions.transactions = true;
+  traceObject.tracerOptions.archiveInterval = 15000;
+  traceObject.tracerOptions.wrapRedis = true;
+  traceObject.tracerOptions.wrapMysql = true;
+  traceObject.tracerOptions.wrapMongodb = true;
+  traceObject.tracerOptions.wrapMemcached = true;
+  traceObject.tracerOptions.wrapPg = true;
+  tracer(traceObject.tracerOptions);
+}
+
+var coreTracer = tracer().tracer;
+agentOptions.strongTracer = coreTracer;
+agentOptions.wrapRedis = coreTracer.config.wrapRedis && USE_PROBES;
```

Comment:
USE_PROBES was removed.  DB probes changes pushed.  The json size increase depends.